content/docs/command-reference/import-url.md

```diff
@@ -55,6 +55,7 @@ DVC supports several types of (local or) remote locations (protocols):
 | -------- | --------------------------------------------------- | ------------------------------------------ |
 | `local`  | Local path                                          | `/path/to/local/data`                      |
 | `s3`     | Amazon S3                                           | `s3://mybucket/data`                       |
+| `azure`  | Microsoft Azure Blob Storage                        | `azure://my-container-name/path/to/data`   |
 | `gs`     | Google Storage                                      | `gs://mybucket/data`                       |
```

Comment:
Unrelated, I know:
```suggestion
| `gs`     | Google Cloud Storage                                | `gs://mybucket/data`                       |
```

Reply:
My plan was to just commit this but the PR doesn't allow maintainers to push to this branch. would be great if you can apply this @steffansluis